manifests/k/Kiwix/Wikivoyage/3.6.6/Kiwix.Wikivoyage.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Kiwix.Wikivoyage
+PackageVersion: 3.6.6
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.14316.0
+InstallerType: appx
+UpgradeBehavior: install
+PackageFamilyName: Kiwix.KiwixJS_5t57d9aw10jep
+RestrictedCapabilities:
+- runFullTrust
+ReleaseDate: 2025-06-09
+AppsAndFeaturesEntries:
+- DisplayName: Kiwix JS
+  Publisher: Association Kiwix
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\WindowsApps\Kiwix.KiwixJS_3.6.6.0_x64__5t57d9aw10jep'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/kiwix/kiwix-js-pwa/releases/download/v3.6.6/Kiwix-JS-Electron-3.6.6-E.appx

```

Comment:
Looking at the previous manifest:

https://github.com/microsoft/winget-pkgs/blob/c7522a04155b80a58683068b9f45e52285e3ebbb/manifests/k/Kiwix/Wikivoyage/3.6.4.0/Kiwix.Wikivoyage.installer.yaml#L13C17-L13C123


This link appears to be incorrect